src/lib/status.ts

```diff
@@ -61,7 +61,7 @@ class StatusBar extends Dispose {
     this.show('restartingLsp...', true);
   }
 
-  private show(message: string, isProgress?: boolean) {
+  show(message: string, isProgress?: boolean) {
```

Comment:
I made this private on purpose,
If we want to show multiple different things on the statusbar but not at the same time we should handle that inside this class, so the show function should remain private

Reply:
Makes sense. How would we show the UI Path though?